docs/getting-started/code-of-conduct.md

```diff
@@ -1,95 +1,96 @@
 # Code of Conduct
 
-This page outlines how you should govern yourself when in our community. If you would like more practical tips and suggestions please read on [how to grow our Community](../community/how-to-grow-our-community.md).
+This page outlines how you should govern yourself when participating in the Status community. For more practical tips and suggestions please refer to [how to grow our Community](../community/how-to-grow-our-community.md).
```

Comment:
ah the reason of using `our` here was deliberate, it's meant to personalise it and be inclusive.